### PR TITLE
Please add githubstudent's pin to the map

### PR DESCRIPTION
**Pull request by just-Meltem**
Adding TR to the map

Closes #950

### CODE REVIEW

**Comment by milenkobrkovic** [comment]
@just-Meltem  This looks great 🎉

File: _pins/github student.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle:  githubstudent
```
Comment (crichID) [inline _pins/github student.json]: @just-Meltem please replace `githubstudent` with your username and check that there is only 1 space between the `:` and your username.

Comment (crichID) [inline _pins/github student.json]: You will need to add a `---` on the first and last line of your file so it looks like this:

```
---
githubHandle: just-Meltem
latitude: 38.614034
longitude: 27.429562
---
```

**Comment by hollenberry** [comment]
Hey @just-Meltem  ! This is good to merge! When you're ready, go ahead and click that big green `Merge Pull Request` button! If you don't do it by the milestone date, we'll go ahead and run that for you in the background.

Let us know if you need any help or have questions! 🎉

**Pull request merged**